Core/Src/main.c

```diff
@@ -87,18 +90,34 @@ int main(void)
   /* Initialize all configured peripherals */
   MX_GPIO_Init();
   MX_USART2_UART_Init();
+  MX_SPI1_Init();
+  MX_TIM1_Init();
   /* USER CODE BEGIN 2 */
 
   /* USER CODE END 2 */
 
   /* Infinite loop */
   /* USER CODE BEGIN WHILE */
-  while (1)
-  {
-    /* USER CODE END WHILE */
+	  uint8_t RxData[3];
+	  uint8_t TxData[3] = {0x01, 0x80, 0x00};
```

Comment:
move to variable def section above. make constant

Reply:
fixed, but I get this warning when I build:
```
../Drivers/STM32F0xx_HAL_Driver/Inc/stm32f0xx_hal_spi.h:794:95: note: expected 'uint8_t *' {aka 'unsigned char *'} but argument is of type 'const uint8_t *' {aka 'const unsigned char *'}
```

Reply:
Oh, I got the order of parameters wrong, fixed.